cmd/kyma/alpha/deploy/cmd.go

```diff
@@ -131,6 +134,11 @@ func (cmd *command) deploy(start time.Time) error {
 		defer func() { os.Stderr = stderr }()
 	}
 
+	clusterInfo, err := clusterinfo.Discover(context.Background(), cmd.K8s.Static())
```

Comment:
Would you kindly define a timeout here and also try to think if you can pass a different ctx here than main?

Reply:
the same discover function is used exactly the same way in the `deploy` and `undeploy` command.
If you think its necessary to change it, than it should be changed all over the place.
I would prefer not to change it as part of this PR

Reply:
Im not worried about the Discover function. I am saying you should probably think of passing `context.WithTimeout(cmd.Context(), INSERT_TIMEOUT_HERE)`.